### DISCUSSION TITLE
OpenWRT路由器无法通过开机自启动文件启动隧道

### DISCUSSION THREAD

**Discussion by WWK-QwQ** [natfrp/wiki#299] [使用问题]
我按照帮助文档安装natfrp并创建自启动文件，但是在输入 /etc/init.d/natfrpc start 后系统日志中提示：
Mon Apr  3 04:01:44 2023 daemon.info natfrpc[5370]: jail: exec-ing /sbin/natfrpc
Mon Apr  3 04:01:44 2023 daemon.info natfrpc[5371]: jail: exec-ing /sbin/natfrpc
Mon Apr  3 04:01:44 2023 daemon.info natfrpc[5370]: jail: jail (5373) exited with exit: 127
Mon Apr  3 04:01:44 2023 daemon.info procd: Instance natfrpc::SakuraFrp s in a crash loop 12 crashes, 0 seconds since last crash
Mon Apr  3 04:01:44 2023 daemon.info natfrpc[5371]: jail: jail (5374) exited with exit: 127
Mon Apr  3 04:01:44 2023 daemon.info procd: Instance natfrpc::SakuraFrp2 s in a crash loop 12 crashes, 0 seconds since last crash

有哪位好心人知道这是怎么回事，该怎么解决

**Comment by fakeboboliu** [comment]
请提供具体信息，如架构、系统版本。
同时尝试执行 `/sbin/natfrpc` 获取回显。

**Comment by WWK-QwQ** [comment]
路由器型号:FriendlyElec NanoPi R2S
架构:arm64
内核版本:5.4.188
固件版本:OpenWrt Rockchip (2023-01-30) / LuCI Master (git-23.026.07704-2908a90)

shell直接运行natfrpc启动和回显正常，运行自启动文件异常

**Reply by fakeboboliu** [reply]
请在我的answer下面使用reply，请把回显给出来，否则我们无法理解你理解的正常。

**Reply by WWK-QwQ** [reply]
好的，回显内容如下(部分隐私内容用*号代替)：

2023/04/07 00:51:18 [I] Checking update...
2023/04/07 00:51:21 [I] frpc version: 0.45.0-sakura-3 (built: 2023-04-02 13:40)
2023/04/07 00:51:21 [I] connecting [frp-fly.top, tcp]
2023/04/07 00:51:22 [I] [72/31979/7d3e] login to server success, get run id [fdb**c4a-7d3e1920]
2023/04/07 00:51:22 [I] [72/31979/7d3e] proxy added: [fdb**c4a.Aliyun_WebDAV fdb**c4a.Yun_WebDAV]
Your HTTP proxy is available now.
Use [**********] to connect.
Don't forget add DNS records to your custom domain
DNS Record value should be: [frp-fly.top]
2023/04/07 00:51:23 [I] [72/31979/7d3e] [fdb**c4a.Aliyun_WebDAV] start proxy success
Your TCP proxy is available now.
Use [**********] to connect.
Or connect by IP (not recommended): [202.182.125.24:65297]
2023/04/07 00:51:23 [I] [72/31979/7d3e] [fdb**c4a.Yun_WebDAV] start proxy success
2023/04/07 00:51:23 [I] [72/31979/7d3e] Speed Limit has been updated: 24 Mibit/s
